src/site/_includes/codelab.njk

```diff
@@ -1,98 +1,91 @@
 ---
-layout: default
+layout: default-next
 permalink: "/{{lang}}/{{page.fileSlug}}/index.html"
 show_banner: false
 pageScripts:
   - '/js/content.js'
 ---
+{% if path === undefined  %}
+  {% set postPaths = page | getPaths %}
```

Comment:
I know this was in the previous code, but I just want to confirm that the single bar is correct here. My understanding is nunjucks uses single bars as filters, but this looks like an "or" expression to me.

Reply:
It should be applied as a filter. I haven't tested this use specifically but I have used it before in a similar manner.